.github/workflows/performance.yml

```diff
@@ -26,4 +26,20 @@ jobs:
       - name: Set up gotestfmt
         uses: GoTestTools/gotestfmt-action@v2            
       - name: Run tests
-        run: go test -v ./pkg/game/performancetests -bench=. -run=xxx -benchmem    
\ No newline at end of file
+        run: go test -v ./pkg/game/performancetests -bench=. -run=xxx -benchmem    
+        
+  request-performance-tests:
```

Comment:
Okay, but there is no benefit compared to the previous version since we are setting up go and building our project twice anyways

Reply:
It can be incorporated into a singe benchmark run